product_docs/docs/biganimal/release/getting_started/creating_a_cluster/creating_an_eha_cluster.mdx

```diff
@@ -35,3 +35,6 @@ After creating the first data group, you can create a second data group for your
 By default, the settings for your first data group populate the second data group's settings. However, if you want to change certain settings you can. Just know that your changes can change the settings for the entire cluster. That being said, the database type and cloud provider must be consistent across both data groups. The data groups and the witness group must all be in different regions. Otherwise, you can choose the second data group's settings as needed. 
 
 When choosing the number of data nodes for the second data group, see [Extreme high availability (Preview)](/biganimal/latest/overview/02_high_availability/#extreme-high-availability-preview) for information on node architecture.
+
+!!! Note
+     If you set maintenance windows for both data groups, make sure they don't overlap. 
```

Comment:
small comment: BA will enforce that they do not overlap - maybe we could say something along the lines of "to maintain high availability, maintenance windows between data groups won't be allowed to overlap"

Reply:
thanks! lgtm